src/yalapack.jl

```diff
@@ -1733,6 +1738,37 @@ function _reorder_realeigendecomposition!(W, WR, WI, work, VR, jobvr)
     end
 end
 
+function _reorder_realeigendecomposition!(W, WR, WI, Wbeta, work, VR, jobvr)
```

Comment:
Possibly can use inbounds annotations in this function, not sure if this matters?

Reply:
It might for large arrays